.argo-ci/ci.yaml

```diff
@@ -41,7 +41,7 @@ spec:
             repo: "{{workflow.parameters.repo}}"
             revision: "{{workflow.parameters.revision}}"
     container:
-      image: argoproj/argo-cd-ci-builder:v0.13.0
+      image: argoproj/argo-cd-ci-builder:v0.14.0
```

Comment:
If you want to keep it matched to release, consider using v0.13.1

Reply:
sure. done